Plugin/Magento/Checkout/Block/Checkout/LayoutProcessor.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+
+namespace Reach\Payment\Plugin\Magento\Checkout\Block\Checkout;
+
+
+class LayoutProcessor
+{
+    /**
+     * Process js Layout of block.
+     * UI item is getting removed
+     * @param \Magento\Checkout\Block\Checkout\LayoutProcessor $subject
+     * @param \Closure $proceed
+     * @param array $jsLayout
+     * @return array
+     */
+    public function aroundProcess($subject, $proceed, $jsLayout)
+    {
+        unset($jsLayout['components']['checkout']['children']['sidebar']['children']['summary']['children']['totals']['children']['grand-total']['config']['basicCurrencyMessage']);
```

Comment:
this is crazy haha thank you for the references in the ticket, that was very helpful. Do you think it would make sense to link to Magento official documentation about this?

although maybe people who work with M2 are used to this, or future devs could just look up the ticket..

Reply:
commit 7773de5ad3c6d1d0c4894180cf3b1d2f9e007757
Thanks!